UPGRADE-15.0.md

```diff
@@ -1045,3 +1045,5 @@ We want to implement more usable UI design which will be better base for upcomin
 
 -   simple navgation images are now blacked-out during cypress tests
 -   make sure you add the blackout everywhere where your snapshots contain simple navigation with images
+
+#### bump next.js and vite version([#3191](https://github.com/shopsys/shopsys/pull/3191))
```

Comment:
You have updated many more packages than just those two, so I think a change of description here and of PR is in order